docs/fidesops/docs/development/fideslog.md

```diff
@@ -0,0 +1,30 @@
+# Fideslog Analytics
+
+Fidesops includes an implementation of [fideslog](https://github.com/ethyca/fideslog) to provide Ethyca with an understanding of user interactions with fides tooling. 
+
+All collected analytics are anonymized, and only used in either product roadmap determination, or as insight into product adoption. Information collected by fideslog is received via HTTPs request, stored in a secure database, and never shared with third parties unless required by law.
+
+More information on use, implementation, and configuration can be found in the [fideslog repository](https://github.com/ethyca/fideslog#readme).
+
+## Collected Data
+Fideslog collects information on instances of fidesops by recording internal events. Starting the server and calling an endpoint will result in sending any or all of the following analytics data to Ethyca:  
```

Comment:
In the future, we'll include not only starting the server and calling endpoints, but also specific interactions from the UI (e.g. toggling show PII, or clicking download report, etc)